### DISCUSSION TITLE
功能请求 (Feature Request) 的 issue 模板中给的 ISSUE.md 的链接似乎挂了

### DISCUSSION THREAD

**Discussion by Eyre-S** [libnyanpasu/clash-nyanpasu#1598] [General]
在尝试创建一个功能请求的issue的时候，去检查下方的自查步骤的时候，点进 ISSUE.md 发现被重定向到了一个 GitHub 上的 404 了的文件。

![image](https://github.com/user-attachments/assets/bc662a7a-f451-4dae-b584-c8f651f98764)

![image](https://github.com/user-attachments/assets/ddc724ab-7f12-423b-8aba-0942c1481fad)  
（被重定向到了这个地方，但是文件 404 了）

看了一下 Bug Report 的模板里的链接是好的，是不是之前迁移这个页面到 WIKI 的时候忘记改这边的链接了？

因为觉得可能不是什么大问题就没发 issue 发 discussion 了。

**Comment by greenhat616** [comment]
PR WELCOME. Thx
